packages/control-property-editor-common/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": ["../../.eslintrc"],
+    "parserOptions": {
+        "EXPERIMENTAL_useSourceOfProjectReferenceRedirect": true,
+        "project": "./tsconfig-test.json"
+    },
+    "overrides": [
```

Comment:
Please remove this override.

Reply:
removed

Reply:
Please also check the `control-property-editor` either remove them there as well or explain why they make sense.

Reply:
I have tried to use in CPE but the app fails to load because its trying to run node js code
```
export const logger = new ToolsLogger({
    transports: [new ConsoleTransport()],
    logPrefix: 'Control-Property-Editor'
});
```

Reply:
<img width="1450" alt="image" src="https://github.com/SAP/open-ux-tools/assets/103586139/72cb6674-f4f0-446e-bdf0-be7079cbb478">

Reply:
overrides are removed.